miniapps/src/main/java/mx/yellowme/youst/miniapps/utils/HapticFeedbackHelper.kt

```diff
@@ -0,0 +1,28 @@
+package mx.yellowme.youst.miniapps.utils
+
+import android.content.Context
+import android.os.Build
+import android.os.VibrationEffect
+import android.os.Vibrator
+
+object HapticFeedbackHelper {
+
+    private const val NEWER_DEVICES_MS = 3L
+    private const val OLDER_DEVICES_MS = 15L
```

Comment:
Maybe both constants could change to an approach more related to its usage:

```
private const val DEFAULT_MILLS = 3L
private const val LEGACY_SUPPORT_MILLS = 15L 
```

or 

```
private const val DEVICES_BELOW_API_26_MILLS = 15L 
```